README.md

```diff
@@ -1 +1,64 @@
-# science-paper-rf-machine-learned-scoring-2022
\ No newline at end of file
+# science-paper-rf-machine-learned-scoring-2022
+
+
+Docking and scoring workflows
+
+---
+
+**Template based docking using the GOLD:**\
+To setup a docking run, generate the following files and folders:
+
+- [docking_home]/tmp_aligned_3d_sdf_sanitized/ligand_templates_for_mcs_manual.sdf \
+contains the manually curated ligand
+templates in 3D. Manual curation of the templates is highly recommended to ensure proper tautomers, avoid low quality
+structures
+- [docking_home]/tmp_aligned_for_MOE_sanitized/*.pdb \
+contains aligned protein structures from MOE projects
+```
+mkdir docking
+cd docking
+bsub < dock.sh
+#after docking has finished
+join_docked_rf_counts.py -t [target]
+```
+---
+**Template based docking in Python**
+`docking.py --input_ligands ligands.sdf -t default -fr=Met713 Met712`
+
+---
+**Template based docking from MOE**
+One can either dock into one of the prepared projects or into an MDB of binding sites.
+The binding sites MDB should have the same structure as a Roche Projects DB.
+If one of the defined projects is selected, the return results will include the RF Score.
+
+---
+
+**Generating project specific parameters:**
+- Series definitions are stored in rf_scoring/series_definitions/[target].json
+
+```
+# Change to your docking directory 
+cd /path/to/docking_dir
+
+# fit RF-PLP
+ccdc_roche_scoring/stat_potential.py -t pde-10
+```
+
+---
+
+**Applications for MOE**
+- Start a webserver:
+```
```

Comment:
I noticed that none of the .py have the typical copyright/licence boilerplate at the top. We typically include these at the CCDC but I guess if the work is done at Roche it might not apply? There should be a LICENSE file in the repo's root.

Reply:
Good point  - have added